modules/calypsoify/style.scss

```diff
@@ -1,8 +1,9 @@
 @import 'colors';
+@import '../../node_modules/@automattic/calypso-color-schemes/dist/calypso-color-schemes.css';
```

Comment:
Note that this isn't a direct dependency yet in `package.json` (it's pulled in via `calypso-build`).

It's being added in https://github.com/Automattic/jetpack/pull/12277 tho.

Reply:
Yup, I noticed that and was planning to add it as well 👍